src/components/app-nav-menu/getting-started/GetStartedPage.tsx

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
What about renaming (GetStartedPage.tsx -> get-started-page.tsx or something like this) ?

Reply:
Please open an issue to agree file naming convention.